classes/trait/driver/field/choices/single.php

```diff
@@ -79,17 +72,11 @@ public function getEmailData($inputValue, Model_Answer $answer)
         // Gets the answer value
         $selectedValue = $this->sanitizeValue($inputValue);
 
-        // Gets the choices list
```

Comment:
nop, `getValueChoiceLabel()` doesn't return the choices list...

If you want a comment here, `Gets the answer label" is better